main.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+function createTable(id, year, month) {
+  const calBody = document.getElementById(`${id}`);
+  const daysInMonthes = [null, 31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31];
+  const nameOfDays = ['ПН', 'ВТ', 'СР', 'ЧТ', 'ПТ', 'СБ', 'ВС'];
+  if(year % 4 === 0) {
+    daysInMonthes[2] = 29;
+  }
+  let days = daysInMonthes[`${month}`];
+  let table = document.createElement('table');
+  calBody.append(table);
+  
+  for (let i = 0; i < 6; i++) {
+    let tr = document.createElement('tr');
+    table.append(tr);
```

Comment:
Always use next logic: create an element with all child, then append the element to the parent